convert_components_to_json.py

```diff
@@ -156,7 +156,65 @@ def convert_components_to_json():
                             component_info["channels"] = 8
                         else:
                             component_info["channels"] = 4
-                
+
+                    # Special handling for GPS over I2C
+                    if component_data.get("isGps", False):
+                            print("**** GPS over I2C detected for component: " + component_dir)
+                            component_info["isGps"] = True
+                    if "commands_ubxes" in component_data:
```

Comment:
this should be accessed under gps.XX, i.e. nested under GPS not directly accessibly at the same object level as isGPS
https://github.com/adafruit/Wippersnapper_Components/compare/main...offline-mode#diff-3dc334943e19fab20af4cd14458687abc17316e6a868ab3097b2eb937e1152c5R89-R96


Same below, L164-169

Probably worth adding a component that tests this functionality to verify it exports correctly, even if only briefly on your local copy.